docs/portal.properties-Reference.md

```diff
@@ -73,11 +73,17 @@ googleplus.consumer.secret=2jCfg4SPWdGfXF44WC588dK
 ```
 (note: these are just examples, you need to get your own) You will also need to go to "Google+ API" and click Enable button. In case of problems make sure to enable DEBUG logging for org.springframework.social and org.springframework.security.web.authentication.
 
+To activate password authentication,
```

Comment:
It seems like there should be a parallel document "Authenticating-and-Authorizing-Users-via-Googleplus.md" rather than showing that method here. Instead I would have put references here to the three/four/five? available guides on how to configure for different authentication methods.
But that change probably belongs in a different PR